dashboard/app/views/gallery_activities/_gallery_activities.html.haml

```diff
@@ -12,5 +12,5 @@
       %div.caption
         =t('gallery.caption_by_name_age', name: user.initial, age: user.age)
     %div.caption
-      =t('gallery.caption_time_ago', time_ago: time_ago_in_words(level_source.created_at))
+      =t('gallery.caption_time_ago', time_ago: time_ago_in_words(gallery_activity.created_at))
```

Comment:
Since we don't generally [update](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/gallery_activities_controller.rb) `GalleryActivity`, I'm not sure that `updated_at` is more meaningful than `created_at`. Let me ping @poorvasingal to find out which she would rather.